src/components/Auth/Alert/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import bellIcon from 'images/icons/bell_icon.png';
+import errorIcon from 'images/icons/error_icon.png';
+
+const Info = ({ title, description }) => {
+  return (
+    <div className="row auth-alert">
+      <div className="col-2">
+        <div className="auth-alert__icon">
+          <img src={ bellIcon } alt="Info" data-testid="info-icon-test" />
```

Comment:
I think we don't need to add some unrelated data-attribute here to test.
What do you think?

Reply:
By unrelated `data-attribute` are you meaning just this `info-icon-test` or every other `data-testid` that I added to test presence of certain element. IMO, user will see this icon so shouldn't we test it's presence?

Reply:
Adding some unrelated data just to test it, doesn't make sense IMO.
You can test the presence of icons without the data attributes I think.

Reply:
As suggested used babel plugin to remove data test attributes from production build.

Reply:
Do we have other way to test it without the data attributes? so we don't need to add this attribute to every element we want to test 🤔

Reply:
@carryall I failed to find any other way with just Jest. Other ways I have seen they usually use Jest along with [Enzyme](https://enzymejs.github.io/enzyme/). But my objective of this certification prefers using just Jest.

Reply:
I somehow missed some docs to test better. Fixed in the new commits.